src/Cli/dotnet/Commands/CliCommandStrings.resx

```diff
@@ -565,6 +565,19 @@ This is equivalent to deleting project.assets.json.</value>
     <value>The directory where the test results will be placed.
 The specified directory will be created if it does not exist.</value>
   </data>
+  <data name="CmdConfigFileDescription" xml:space="preserve">
+    <value>Specifies a testconfig.json file.</value>
+  </data>
+  <data name="CmdConfigFilePath" xml:space="preserve">
+    <value>CONFIG_FILE</value>
+  </data>
+  <data name="CmdDiagnosticOutputDirectoryDescription" xml:space="preserve">
+    <value>Output directory of the diagnostic logging.
+If not specified the file will be generated inside the default 'TestResults' directory.</value>
```

Comment:
it's confusing that the first sentence talks about directory, whereas the second sentence talks about a file.